src/main/g8/src/main/scala/com/example/routes/SimpleRoutes.scala

```diff
@@ -0,0 +1,19 @@
+package com.example.routes
+
+import akka.http.scaladsl.marshallers.xml.ScalaXmlSupport
+import akka.http.scaladsl.server.directives.MethodDirectives.get
+import akka.http.scaladsl.server.directives.PathDirectives.path
+import akka.http.scaladsl.server.directives.RouteDirectives.complete
+
+/**
+  * Routes can be defined in separated classes like shown in here
+  */
+trait SimpleRoutes extends ScalaXmlSupport {
```

Comment:
We should maintain the best practice not to extend from the marshalling support classes but to import from the object instead. This keeps scopes clean from potential conflicting implicit definitions.